CONTRIBUTING.md

```diff
@@ -96,6 +122,11 @@ executed  through GitHub Actions after each commit, during pull requests,
 and daily. The tests connect to a private CashCtrl test account with the API
 key stored as a GitHub secret.
 
+Before you try to run tests, execute the following command, to install pytest
```

Comment:
line 125 can be shortened: ~~Before you try to run tests,~~ Execute the following command to install pytest:

Reply:
Fixed